src/main/java/ru/training/at/hw8/constants/SingleValueParameter.java

```diff
@@ -0,0 +1,19 @@
+package ru.training.at.hw8.constants;
+
+import ru.training.at.hw8.propertyloader.PropertyLoader;
+
+import java.util.Properties;
+
+import static ru.training.at.hw8.propertyloader.PropertyLoader.*;
+
+public class SingleValueParameter {
+    private static final Properties envProperties = getProperties();
+    public static final String KEY;
+    public static final String TOKEN;
+
+    static {
+        assert envProperties != null;
```

Comment:
Why do you have 'assert' here?

Reply:
Because there was warning that it can throw NullPointer